### PR TITLE
Remove print statements for loop start and end

### PR DESCRIPTION
**Pull request by avdm12-elementary**
## Change description

In metrics_monitory.py, we replace print statements for "Monitor loop start" and "Monitor loop end" with debug logs, and replace the "ready and beginning" print statement with an info log.

In atom.c, we print only when the severity level is equal to or more severe than info.

Validation:

Confirmed that the print statements were replaced with no apparent issues.

## Type of change
- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by aywaldron — CHANGES_REQUESTED** [review]
Hey @avdm12-elementary , I just realized there is actually an atom `element` created in the main loop of this function which we could use for logging instead of print statements. Might be good to change these to log through the atom element at debug level, e.g. `element.log(LogLevel.DEBUG, ...)`.

File: metrics/monitoring/metrics_monitor.py [hunk 1]
```diff
@@ -626,8 +626,6 @@ def initialize_timing_metric(metric):
     # Loop forever
```
Comment (aywaldron) [inline metrics/monitoring/metrics_monitor.py:626]: We can change the print statement at line 624 to be logged via atom as well, at INFO level

**Review by aywaldron — APPROVED** [review]
LGTM, thanks for getting this up!

**Review by dpipemazo — APPROVED** [review]
Thank you!

**Pull request merged**